rust_src/src/image.rs

```diff
@@ -1,10 +1,11 @@
 //! Functions for image support on window system.
 
-#[cfg(feature = "glyph-debug")]
 use remacs_macros::lisp_fn;
 
 #[cfg(feature = "glyph-debug")]
-use crate::{lisp::LispObject, remacs_sys::valid_image_p};
+use crate::remacs_sys::valid_image_p;
+
+use crate::{lisp::LispObject, remacs_sys::lookup_image_type};
```

Comment:
group the `remacs_sys items together please.
This should be:

```
use crate::lisp::LispObject;
use crate::remacs_sys::{valid_image_p, lookup_image_type};
```

Reply:
Even if that means moving the `valid_image_p` out from under the `#[cfg(feature = "glyph-debug")]`?

Reply:
If the `glyph-debug` remains then import it separately. Maybe that is why the two other imports were grouped together? Was that done by you or by rustformat?

Reply:
I grouped the imports as they are currently. `lisp::LispObject` was needed by both `init_image_library` and `imagep` but `valid_image_p` was only used by `imagep`, which is under `#[cfg(feature = "glyph-debug")]`. It felt right to only import `valid_image_p` in the scenarios where it is needed. Happy to expose everything though if thats the preference.

Reply:
```
use crate::lisp::LispObject;
use remacs_sys::lookup_image_type;
```

Then see what rustformat does with it.....